lib/web_ui/lib/src/engine/canvaskit/text.dart

```diff
@@ -87,6 +89,17 @@ class CkParagraphStyle implements ui.ParagraphStyle {
       skStrutStyle.heightMultiplier = style._height;
     }
 
+    switch (style._leadingDistribution ?? paragraphHeightBehavior?.leadingDistribution) {
```

Comment:
nit: I'd extract the switch expression into a variable. Typically we use "effective" as the prefix for something that's resolved from multiple sources, e.g. `effectiveLeadingDistribution`.